app/helper/rtForm.php

```diff
@@ -236,6 +240,7 @@ private function generate_element_value( $element, $attributes ) {
 				case 'rtHidden': //hidden
 				case 'rtNumber': //number
 				case 'rtText' : //text
+				case 'rtButton': //button
 					$html .= 'value="';
 					$html .= esc_attr( ( isset( $attributes['value'] ) ) ? $attributes['value'] : '' );
```

Comment:
We don't need to escape empty value. Please change this to `$html .= ( isset( $attributes['value'] ) ) ? esc_attr( $attributes['value'] ) : '';`